lago/cmd.py

```diff
@@ -271,6 +271,39 @@ def do_stop(prefix, vm_names, **kwargs):
     prefix.stop(vm_names=vm_names)
 
 
+@lago.plugins.cli.cli_plugin(help='Export virtual entities')
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--entities-names',
+    '-e',
+    help='Name of the entities to export. If no name is specified, export all'
+    'the entities in this prefix',
+    metavar='ENTITY_NAME',
+    nargs='*',
+)
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--standalone',
+    '-s',
+    help='If not specified, export a layered image',
+    action='store_true'
+)
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--compress',
+    '-c',
+    help='If specified, compress the exported images',
+    action='store_true'
```

Comment:
might be better to use 'store' and give it a default of ``True`` so it could be configurable in lago.conf
(we can't configure 'store_true' ATM)